app/views/search/index.html.erb

```diff
@@ -1,6 +1,8 @@
 <div class="container">
   <h1 class="my-4" id="results-header">Community Search Results for <%= @name %>:</h1>
   <input type="button" id="change" value="View NYTimes Results" class="btn btn-outline-secondary mr-1">
+  <br>
```

Comment:
I think it looks a little cleaner with these spaces.

Reply:
![](https://media0.giphy.com/media/xT8qBhrlNooHBYR9f2/giphy.gif?cid=5a38a5a2twbuirtzaxnwuzncauqng0xzrg80564b146yk50l&rid=giphy.gif)